rts/Rendering/Env/IGroundDecalDrawer.h

```diff
@@ -13,6 +13,8 @@ class IGroundDecalDrawer
 	CR_DECLARE(IGroundDecalDrawer)
 public:
 	static bool GetDrawDecals() { return (decalLevel > 0); }
+	static int GetDecalLevel() { return decalLevel; }
```

Comment:
Shouldn't this return `std::abs(decalLevel)`? Since level `+x` means x, but `-x` also means x, just currently disabled.

Reply:
> I was inclined to remove altogether or just make it binary.

Yeah maybe the current "multiplier for decal duration" doesn't make much sense (see also #1476). But some way to control how many decals there can be is good imo (as a performance/visuals knob, since a battlefield littered with scars is cool which is the original reason to extend decal duration).

If not touching the design then perhaps best to just be clean and have separate bool and (u?)int instead of doing bit hax to save 4 bytes. Anyway, current is also fine (since it's just used by the debug print in the `/decals` command).